pages/index.vue

```diff
@@ -0,0 +1,519 @@
+                </p>
+                <div class="mt-11 flex items-center justify-center">
+                    <button class="btn-primary--large">
+                        Candidati subito
+                    </button>
+                </div>
+            </div>
+            <div class="flex items-center justify-center flex-wrap  mt-12 px-4 lg:px-0">
+                <img src="/hero-img-1.png" alt=" hero img-1" class="lg:-mr-24 lg:mt-20 lg:z-30" data-aos="zoom-in"
+                    data-aos-delay="50">
+
+                <img src="/hero-img-2.png" alt=" hero img-1" class="lg:-mr-10 lg:z-20" data-aos="zoom-in"
+                    data-aos-delay="100">
+
+                <img src="/hero-img-3.png" alt=" hero img-1" class="lg:-mr-16 lg:mt-10 lg:z-10" data-aos="zoom-in"
+                    data-aos-delay="150">
+
+                <img src="/hero-img-4.png" alt=" hero img-1" data-aos="zoom-in" data-aos-delay="200">
+            </div>
+        </div>
+    </section>
+
+    <!-- project section -->
+    <section class="mt-40 lg:mt-[188px]">
+        <div class="container relative" data-aos="fade-up">
+            <p class="text-white text-3xl md:text-6xl">
+                Il Padel World Club DAO <br>
+                mira a
+                <span class="inline-flex items-center gap-1 bg-secondary text-app-bg px-2 my-1">
+                    rivoluzionare
+                    <icon-rotate />
+                </span> <br>
+
+                l'esperienza del
+                <span class="inline-flex items-center gap-1 bg-secondary text-app-bg px-2 my-1">
+                    padel
+                    <icon-livello />
+                </span>
+                creando <br>
+                un circuito elitario di club <br>
+                diffusi <span class="inline-flex items-center gap-1 bg-secondary text-app-bg  px-2 my-1">
+                    globalmente
+                    <icon-globe />
+                </span> attraverso <br>
+                la tecnologia web3.
+            </p>
+            <span
+                class=" text-app-bg bg-[#D3E2E8] px-2 md:px-4 py-2 text-xl md:text-3xl absolute  -top-20 left-6 md:-left-2 rotate-[-5.45deg]">IL
+                PROGETTO</span>
+        </div>
+    </section>
+
+    <!-- marque section -->
+    <section class="mt-40 lg:mt-[131px] pt-11 pb-16 border border-white border-l-0 border-r-0 border-opacity-25">
+        <vue3-marquee :clone="true" :duration="20" :direction="'reverse'" :pause-on-hover="true"
+            @onOverflowCleared="alert()" class=" overflow-y-hidden">
+            <div class="text-white font-medium text-4xl md:text-6xl lg:text-[86px] flex items-center gap-5 italic hover:text-[#4D696D] hover:not-italic mr-10 transition duration-200 ease-linear"
+                v-for="text in marqueText" :key="text">
+                {{ text }}
+                <icon-arrow-right-large class="w-12 h-12 md:w-24 md:h-24 lg:w-[91px] lg:h-[84px]" />
```

Comment:
here the same instead to use `lg:w-[91px]` you can use `w-24` which is 96px.

be careful to put height different from width, very often the icons have height = width or it is better to put only one of the two to avoid problems